scripts/build.sh

```diff
@@ -611,7 +611,7 @@ else
 fi
 cp ../installer/Install.ps1 "$WORK_DIR/wsa/$ARCH" || abort
 cp ../installer/Run.bat "$WORK_DIR/wsa/$ARCH" || abort
-find "$WORK_DIR/wsa/$ARCH" -maxdepth 1 -mindepth 1 -printf "%P\n" >"$WORK_DIR/wsa/$ARCH/filelist.txt" || abort
+find "$WORK_DIR/wsa/$ARCH" -printf "%P\n" | sed -e 's/\//\\/g' >"$WORK_DIR/wsa/$ARCH/filelist.txt" || abort
```

Comment:
Why you change to this ones @s1204IT

Reply:
Yeah,
I've found a flaw and are correcting it.